pkg/app/execontext.go

```diff
@@ -197,13 +229,41 @@ func (ref *Output) Info(infoType string, params ...OutVars) {
 		}
 	}
 
-	fmt.Printf("cmd=%s info=%s%s%s\n", ref.CmdName, itcolor(infoType), sep, data)
+	switch ref.JSONFlag {
+	case "json":
+		var jsonData []byte
+		if len(data) > 0 {
+			jsonData, _ = json.Marshal(params[0])
+			fmt.Println(string(jsonData))
+		}
+	case "text":
+		fmt.Printf("cmd=%s info=%s%s%s\n", ref.CmdName, itcolor(infoType), sep, data)
+
+	default:
+		fmt.Printf("Unknown json flag: %s\n", ref.JSONFlag)
+	}
+
 }
 
-func ShowCommunityInfo() {
+func ShowCommunityInfo(jsonFlag string) {
+
+	type Data struct {
+		DockerSlim string `json:"dockerslim`
+	}
+
+	var data Data
+
 	color.Set(color.FgHiMagenta)
 	defer color.Unset()
-	fmt.Printf("docker-slim: message='join the Gitter channel to ask questions or to share your feedback' info='%s'\n", consts.CommunityGitter)
-	fmt.Printf("docker-slim: message='join the Discord server to ask questions or to share your feedback' info='%s'\n", consts.CommunityDiscord)
-	fmt.Printf("docker-slim: message='Github discussions' info='%s'\n", consts.CommunityDiscussions)
+	data.DockerSlim = fmt.Sprintf("message='join the Gitter channel to ask questions or to share your feedback info='%s'"+"\n"+"message='join the Discord server to ask questions or to share your feedback' info='%s'"+"\n"+"message='Github discussions' info='%s'", consts.CommunityGitter, consts.CommunityDiscord, consts.CommunityDiscussions)
```

Comment:
This can be simplified a bit... The json equivalent of the output should have two fields: `app` and `message`. We already have the `message` field in the text output. We need to add the `app` field, so the output looks a bit nicer (here's a snippet to show what i mean):

`{"app":"docker-slim", "message":"join the Gitter channel..."}`

Reply:
Thanks so much, have tried to fix this, please review